### DISCUSSION TITLE
Does it work on Mobile?

### DISCUSSION THREAD

**Discussion by codey-badger** [gasparschott/obsidian-continuous-mode#24] [Q&A]
Thanks for the plugin, it looks awesome in the documentation.  

My motive, as the title says, is to know if it works properly on mobile. As  'spliting' already doesn't do anything noticeable for me on my Vault and I don't see any information about mobile functions on the Readme section. In either  case, please, consider add the information or let me know if I should open an issue instead. 

TIA

**Comment by gasparschott** [comment]
TBH, I haven't done as much testing on mobile as I should. The basic functionality is available, but it seems the Continuous Mode submenus don't appear on touch. Can you confirm this and any other problems you've encountered? I can't say how quickly I'll be able to address them, but I definitely want the plugin to work as well as it can on mobile. Thanks for your inquiry.

**Reply by Qloulia** [reply]
I searched Scrivener and found your plug-in, very good stuff you got going on here! Unfortunately it seems not working on the mobile right now, I'm looking forward for this!

**Reply by marellec** [reply]
I'm wondering if you are still working on this for mobile?
I would really love to use continuous mode, but it interferes with the ... menu for each note, so that clicking any menu option triggers a different menu option.

**Comment by gasparschott** [comment]
Thanks for the report, I'm afraid I haven't been doing any testing on mobile for a while. I will look into the situation when I have time, though IIRC the problem lies with limitations for adding items to contextual menus on mobile. Most of the menu items can be invoked from the command palette—would that help?

Sent with [Proton Mail](https://proton.me/mail/home) secure email.

**Reply by marellec** [reply]
No problem! Even though the issue is still there, I am using a theme called Prsim which lets you scroll the menu, and that helps recalibrate the touch input.

And you are right, thanks, I can use the command palette if it stops working!
